### PR TITLE
Allow Precland fast descend

### PR DESCRIPTION
**Pull request by rishabsingh3003**
This allows user to skip slowing down the speed of the copter automatically when the vehicle is 2m above the target.

### CODE REVIEW

File: libraries/AC_PrecLand/AC_PrecLand.cpp [hunk 1]
```diff
@@ -178,7 +178,7 @@ const AP_Param::GroupInfo AC_PrecLand::var_info[] = {
     // @Param: OPTIONS
     // @DisplayName: Precision Landing Extra Options
     // @Description: Precision Landing Extra Options
-    // @Bitmask: 0: Moving Landing Target, 1: Allow Precision Landing after manual reposition 
+    // @Bitmask: 0: Moving Landing Target, 1: Allow Precision Landing after manual reposition, 2: Do not slow vertical speed in final descend phase
```
Comment (rmackay9) [inline libraries/AC_PrecLand/AC_PrecLand.cpp:181]: nitpick: maybe we could word this in a positive way?  E.g. "Maintain high speed in final descent".

Also I think "descent" is better than "descend".

**Review by rmackay9 — APPROVED** [review]
Functionally I think this is OK, just some minor naming changes and then it's fine to merge, thanks.

**Comment by rishabsingh3003** [comment]
Rebasing for unrelated CI failures

**Comment by peterbarker** [comment]
> Rebasing for unrelated CI failures. I think I pressed the wrong button on the "github rebase"...

You did - leaving a merge commit in place.

I think you can probably press more buttons in the interface to fix that, but I just rebased it by hand.

**Comment by rishabsingh3003** [comment]
Thanks Peter!

**Pull request merged**